components/google_sheets/sources/new-row-added/new-row-added.js

```diff
@@ -87,12 +102,12 @@ module.exports = {
     },
   },
   methods: {
-    getMeta(spreadsheet, sheet, diff) {
+    getMeta(spreadsheet, sheet) {
       return {
         id: `${spreadsheet.spreadsheetId}${
           sheet.properties.sheetId
         }${Date.now()}`,
-        summary: `${diff} rows added to ${spreadsheet.properties.title} - ${sheet.properties.title}`,
+        summary: `New row added to ${spreadsheet.properties.title} - ${sheet.properties.title}`,
```

Comment:
The summary will be the same for every row now, so maybe we do something like:

```
summary: data.join(", ")
```

where `data` is the array of new row data?